components/editor/ai-integration/overview.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Overview
+page_title: Editor AI Integration Overview
+description: Overview of the AI integration options that the Editor for Blazor provides.
+slug: editor-ai-integration-overview
+tags: telerik,blazor,editor,ai,ai integration, aiprompt
+published: True
+position: 0
+---
+
+# AI Integration Overview
+
+The Editor provides AI integration to enhance the user productivity and streamline workflows. This integration is available in two ways:
+
+* **AIPrompt Component**: An [AIPrompt component](slug:aiprompt-overview) that is rendered in a separate pane on the left side of the Editor.
+
+* **Inline Prompt**: A lightweight, on-the-fly AI prompt rendered in a popup that is shown once the user selects text in the Editor.
```

Comment:
"on-the-fly" doesn't carry enough meaning and the rest of the text should be clear enough.

```suggestion
* **Inline Prompt**: A lightweight AI prompt rendered in a popup that is shown once the user selects text in the Editor.
```